script.js

```diff
@@ -0,0 +1,43 @@
+const button = document.querySelector('button');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const content = document.querySelector('.books');
+
+const userData = JSON.parse(localStorage.getItem('formdata')).reverse();
+let count = userData.length;
+
+/* eslint no-unused-vars: "off" */
+function deleteItem(author) {
+  const filteredArray = userData.filter((e) => e.author !== author);
+  localStorage.setItem('formdata', JSON.stringify(filteredArray));
+}
```

Comment:
[OPTIONAL]
- [ ] It would be better if you may consider deleting a book through a unique identifier as it can happen that many books have one author.
hint: YOu may consider setting a `id` for Each added book and using it once deleting a specific book. YOu may consider doing this is your addbook function:
```js
const books = [];

const addBook = (title, author) => {
  id = generateID(); `You may say #{id = Date.now()}`
  books = books.concat({ title, author, author });
};
```